billing.md

```diff
@@ -1314,7 +1314,7 @@ Of course, you may also cancel the subscription entirely:
 
```

Comment:
I'd rename this entire section to "Usage-Based Billing which is exactly how Stripe calls it.